.pack.mk

```
@@ -22,17 +22,28 @@ ifeq (${VERSION}, $(filter ${VERSION}, 1.10))
 endif
 
 prepare:
-	rm -rf build packpack
-	git clone https://github.com/packpack/packpack.git
+#	rm -rf build packpack
+#	git clone https://github.com/packpack/packpack.git
 
 package: prepare
 	if [ "${TARANTOOL_SERIES}" != "1.10" ]; then \
 		if [ -n "${GIT_TAG}" ]; then \
 			export VERSION="$$(echo ${GIT_TAG} | sed 's/-/~/')"; \
 		else \
-			export VERSION="$$(echo ${GIT_DESCRIBE} | sed ${SED_REPLACE_VERSION_REGEX} | sed 's/-/~/').dev"; \
+			if [ "${OS}" = "alpine" ]; then \
+				if git describe | grep -q "entrypoint" ; then \
+					# Set VERSION to format x.x.x_alapa0 https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#pkgver \
```

does the comment need a line wrap?

```suggestion
					# Set VERSION to format x.x.x_alpha0 https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#pkgver \
```

---

I removed this variable, plan has changed.

---

okay